pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -485,6 +486,25 @@ public void testLiteralOnlyFunc()
     assertEquals(results.get(10).asText(), "hello!");
   }
 
+  @Test
+  public void testMultiValueColumnGroupBy()
+      throws Exception {
+    String pinotQuery = "SELECT count(*), arrayToMV(RandomAirports) FROM mytable "
+        + "GROUP BY arrayToMV(RandomAirports)";
+    JsonNode jsonNode = postQuery(pinotQuery);
+    Assert.assertEquals(jsonNode.get("resultTable").get("rows").size(), 154);
+  }
+
+  @Test
+  public void testMultiValueColumnGroupByOrderBy()
+      throws Exception {
+    String pinotQuery = "SELECT count(*), arrayToMV(RandomAirports) FROM mytable "
```

Comment:
will it work if I run 
```
SELECT count(*), arrayToMV(RandomAirports) 
FROM mytable 
WHERE Dest IN (SELECT Dest FROM myTable GROUP BY Dest HAVING count(*) > 10)
```
(later when we implemented the scalar function wrapper)

Reply:
You need a `group by arrayToMV(RandomAirports)` ?

Reply:
This only works at leaf stage not intermediate stage.

Reply:
Ok sounds good